traefik/templates/requirements.yaml

```
@@ -93,4 +93,7 @@
     {{ fail "ERROR: offline mode is a _licensed_ feature only available for Traefik Hub >= v3.18.0."}}
   {{- end }}
 
+  {{- if and (semverCompare "<v3.18.0" $hubVersion) .Values.hub.mcpgateway.enabled }}
```

```suggestion
  {{- if and (semverCompare "<v3.18.0-0" $hubVersion) .Values.hub.mcpgateway.enabled }}
```
You need this in order for it to work with rc